static/images/logos/funders/reverse/concordia-logo.html

```diff
@@ -0,0 +1,117 @@
+
+<html>
```

Comment:
This is an entire html block ... this can't be right?

Reply:
I'm experimenting turning this into an svg image by using Inkscape's xml editor.

Reply:
Concordia logo converted to svg.